### PR TITLE
Power init, perfd, SDcardFS & missing properties/permissions.

### PR DESCRIPTION
**Pull request by danielstuart14**
Changes:
-Replaced post_boot script with CAF qcom.power init.
-Enabled core control.
-Enabled SDcardFS
-Added perfd service. (MSM power management solution)
-Added missing properties
-Added missing permissions
-Cleaned device init

### CODE REVIEW

File: proprietary-files.txt [hunk 1]
```diff
@@ -169,6 +169,10 @@ etc/firmware/a300_pm4.fw
 etc/firmware/a420_pfp.fw
 etc/firmware/a420_pm4.fw
 
+# Perf
```
Comment (Shr3ps) [inline proprietary-files.txt]: P comes after M :p

Comment (danielstuart14) [reply proprietary-files.txt]: oh god, that's embarrassing

**Comment by chaubeyprateek** [comment]
Core control is not required...I have already tested it on my other devices and result were meh...

**Comment by danielstuart14** [comment]
Well, CAF msm89x9 uses it. It seems right to follow that.
We don't want all 8 cores enabled at all the time, the 2 cluster architecture was projected in an way that you should only use the performance cluster when needed.

**Comment by karthick111** [comment]
@danielstuart14  close this PR 
and check this https://github.com/boulzordev/android_device_lenovo_A6020/pull/24

**Pull request merged**